### DISCUSSION TITLE
Recipe for using browser-level interactivity

### DISCUSSION THREAD

**Discussion by mx781** [idom-team/idom#504] [Question] [answered]
I'm wondering if there's a way to make use of browser-level interactivity in IDOM, i.e. `alert()`, `prompt()` and `confirm()`. The use case I have in mind is to make use of confirm() as a guard against unintended clicking of a button that would otherwise cause an event to fire. From what I've seen, executing/rendering arbitrary JS is only possible via custom JS modules. Is there a simpler solution for this, or is my best bet to create the dialog and ok/cancel buttons myself?

**Comment by rmorshea** [comment, marked as answer]
No. Presently it is not possible to invoke browser-level interactivity without creating custom JS components. With that said, for things like `alert()`, `prompt()`, and `confirm()` IDOM should supply basic implementations so users like yourself don't have to create their own.
